tensornetwork/backends/abstract_backend.py

```diff
@@ -988,3 +988,16 @@ def deserialize_tensor(self, s: str) -> Tensor:
     raise NotImplementedError(
         "Backend '{}' has not implemented deserialize_tensor.".format(
             self.name))
+
+  def power(self, a: Tensor, b: Tensor) -> Tensor:
```

Comment:
b should be allowed to be a scalar as well, in which case the output should be each element in a raised to b.